.github/workflows/FissionESLintFormat.yml

```diff
@@ -24,30 +24,14 @@ jobs:
         npm install
     - name: Linter
       id: linter-validation
+      if: ${{ always() }}
       run: |
         cd fission
-        npm run lint
-      continue-on-error: true
-    - name: Check EsLint
-      run: |
-        if [ ${{ steps.linter-validation.outcome }} == "success" ]; then
-          echo "EsLint Validation Passed"
-        else
-          echo "EsLint Validation Failed"
-          exit 1
-        fi
+        npm run lint && echo "ESLint Validation Passed" || (echo "ESLint Validation Failed" && exit 1)
     - name: Prettier
       id: prettier-validation
+      if: ${{ always() }}
```

Comment:
Is the if always really necessary? We don't use it in our other workflows, would like to keep them the same if possible.